ui/src/app/subnets/views/SubnetDetails/UsedIPs/SubnetUsedIPs.tsx

```diff
@@ -1,151 +0,0 @@
-import { MainTable, Spinner } from "@canonical/react-components";
-import { useSelector } from "react-redux";
-
-import TitledSection from "app/base/components/TitledSection";
-import type { RootState } from "app/store/root/types";
-import subnetSelectors from "app/store/subnet/selectors";
-import type {
-  Subnet,
-  SubnetDetails,
-  SubnetIP,
-  SubnetMeta,
-} from "app/store/subnet/types";
-import { IPAddressType, IPAddressTypeLabel } from "app/store/subnet/types/enum";
-import { NodeType, NodeTypeDisplay } from "app/store/types/node";
-
-export type Props = {
-  subnetId: Subnet[SubnetMeta.PK] | null;
-};
-
-export enum Labels {
-  IpAddresses = "IP addresses",
-  Type = "Type",
-  Node = "Node",
-  Interface = "interface",
-  Usage = "Usage",
-  Owner = "Owner",
-  LastSeen = "Last Seen",
-}
-
-const getType = (ip: SubnetIP) => {
-  return IPAddressTypeLabel[
-    IPAddressType[ip.alloc_type] as keyof typeof IPAddressTypeLabel
```

Comment:
@andesol as mentioned above, using an enum for the labels means we have to explicitly cast the type here, which I almost always try to avoid.